docs/labs/aws/kmm-kmod.yaml

```diff
@@ -9,43 +9,14 @@ spec:
         moduleName: kmm-ci-a 
 
       kernelMappings:
-        - literal: 5.4.196-108.356.amzn2.x86_64
-          containerImage: <aws_account_number>.dkr.ecr.<aws_region>.amazonaws.com/kmmo-lab:kmm-2.2
+        - regexp: '^.+.amzn2' 
```

Comment:
```suggestion
        - regexp: '^.+-amzn2$' 
```

Reply:
This won't be modified as regexp would not match with kernel version string.